internal/build/path_mapping.go

```diff
@@ -132,7 +137,11 @@ func fileToPathMapping(file string, sync []model.Sync) (PathMapping, *PathMappin
 			}, nil
 		}
 	}
-	return PathMapping{}, pathMappingErrf("file %s matches no syncs", file)
+	// (Potentially) expected case: the file doesn't match any sync src's. It's up
```

Comment:
talked about this offline...this seems very dangerous, and has bitten us in the past (which is why we had the error in the first place)